src/style.css

```
@@ -0,0 +1,116 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 700;
+}
+
+:root {
+  --light-shadow: rgba(61, 78, 97, 0.1);
+  --dark-shadow: rgba(61, 78, 97, 0.2);
+  --gradient: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+}
+
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Avenir", sans-serif, monospace;
+}
+
+.search {
+  display: flex;
+  width: 100%;
+  justify-content: center;
+  margin: 20px auto 0;
+  height: 70px;
+}
+
+.search_small {
+  height: 42px;
+}
+
+.search__label {
+  position: relative;
+  display: block;
+  width: 100%;
+}
+
+.search__label:after {
+  position: absolute;
+  content: "";
+  display: block;
+  background-image: url("./images/Search.svg");
+  background-repeat: no-repeat;
+  background-size: contain;
+}
+
+.search__label_big:after {
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.search__label_small:after {
+  width: 11px;
+  height: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.search__input {
+  width: 100%;
+  height: 100%;
+  font-family: inherit;
+  font-weight: 300;
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px var(--light-shadow);
+  outline: none;
+}
+
+.search__input_big {
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.search__input_small {
+  padding-left: 33px;
+  padding-top: 0;
+  font-size: 14px;
+  line-height: 19px;
+}
```

As suggested before you can add big form/label/input styles to the common selector, and add only one modifier for small
```suggestion
.search__input {
  width: 100%;
  height: 100%;
  font-family: inherit;
  font-weight: 300;
  color: #3d4e61;
  border: 1px solid #e1e7ed;
  border-radius: 4px;
  box-shadow: 0 1px 8px var(--light-shadow);
  outline: none;
  padding-left: 62px;
  font-size: 16px;
  line-height: 22px;
}

.search__input_small {
  padding-left: 33px;
  padding-top: 0;
  font-size: 14px;
  line-height: 19px;
}
```

Check for other selectors, you only need one modifier for small form, no need for creating big modifier